### PR TITLE
Init

### PR DESCRIPTION
**Pull request by jchavez443**

### CODE REVIEW

File: .github/actions/maven-test/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Container image that runs your code
```
Comment (jmulford-bw) [inline .github/actions/maven-test/Dockerfile:1]: I thought we decided no tests for sample apps?

**Review by jmulford-bw — APPROVED** [review]
Just 1 question, looks good otherwise

File: .github/actions/maven-test/Dockerfile (hunk 1 above)
Comment (jchavez443) [reply .github/actions/maven-test/Dockerfile:1]: Its a compile test.  Its just easy to copy and past the code over.  Its not really need but I figured it couldn't hurt.

Comment (jmulford-bw) [reply .github/actions/maven-test/Dockerfile:1]: Ok cool that's fine then

**Review by hamermike — APPROVED** [review]

**Pull request merged**